corehq/apps/builds/utils.py

```diff
@@ -40,3 +40,47 @@ def extract_build_info_from_filename(content_disposition):
     else:
         raise ValueError('Could not find filename like {!r} in {!r}'.format(
             pattern, content_disposition))
+
+
+def get_latest_version_at_time(target_time):
+    """
+    Get the latest CommCare version that was available at a given time.
+    Excludes superuser-only versions.
+    Menu items are already in chronological order (newest last).
+    If no target time is provided, return the latest version at now.
```

Comment:
change to: return the latest version available now

Reply:
a65c782bd982626d2c8526be4125bde6a5b0d26b